wm/frame.go

```diff
@@ -3,271 +3,214 @@
 package wm
 
 import (
-	"log"
-
 	"fyne.io/fyne"
 	"fyne.io/fyne/app/util"
 	"fyne.io/fyne/test"
 	"fyne.io/fyne/theme"
 
 	"github.com/BurntSushi/xgb/xproto"
-	"github.com/BurntSushi/xgbutil/icccm"
-	"github.com/BurntSushi/xgbutil/xevent"
-	"github.com/BurntSushi/xgbutil/xprop"
 	"github.com/BurntSushi/xgbutil/xwindow"
 
 	"fyne.io/desktop"
 	wmTheme "fyne.io/desktop/theme"
 )
 
 type frame struct {
-	id, win        xproto.Window
-	x, y           int16
-	width, height  uint16
-	mouseX, mouseY int16
-	mouseResize    bool
+	cli                         *client
+	x, y                        int16
+	restoreX, restoreY          int16
+	width, height               uint16
+	restoreWidth, restoreHeight uint16
+	mouseX, mouseY              int16
+	mouseResize                 bool
 
 	framed              bool
 	minWidth, minHeight uint
-	title               string
 
-	wm     *x11WM
 	canvas test.WindowlessCanvas
 }
 
-func (s *stack) frameForWin(id xproto.Window) desktop.Window {
-	for _, w := range s.frames {
-		if w.(*frame).id == id || w.(*frame).win == id {
-			return w
-		}
-	}
-
-	return nil
-}
-
-func (f *frame) unFrame() {
-	if !f.framed {
-		return
-	}
-
-	frame := f.wm.frameForWin(f.win)
-	f.wm.RemoveWindow(frame)
-
-	if frame == nil {
+func (fr *frame) unFrame() {
```

Comment:
Why the change from "f" to "fr"? it's not common and is making the diff bigger than it needs to be.